docs/userguides/accounts.md

```diff
@@ -234,6 +234,23 @@ message = encode_defunct(text="Hello Apes!")
 signature = account.sign_message(message)
 ```
 
+## Signed Message
+
+Get the signed message signature as well as message hash signed by the account
+
+```py
+from ape.types.signatures import recover_signer
+
+[...]
+signature = account.sign_message(message)
+
+# signature
+print(signature.encode_vrs())
+# message hash
+print(signature.message_hash)
+
```

Comment:
```suggestion
```

Reply:
Can you suggest what examples I should put here ?